src/mito2/src/region_write_ctx.rs

```diff
@@ -92,6 +94,10 @@ pub(crate) struct RegionWriteCtx {
     ///
     /// The i-th notify is for i-th mutation.
     notifiers: Vec<WriteNotify>,
+    /// Notifiers for bulk requests.
+    bulk_notifiers: Vec<WriteNotify>,
+    /// Pending bulk write requests
+    pub(crate) bulk_parts: Vec<BulkPart>,
```

Comment:
Maybe we should replace WriteRequest with an enum so we can support bulk/normal in the same struct in the future.

Now we have to copy-paste similar codes: WriteNotify, pending requests, stalled requests......